spec/mail/elements/date_time_element_spec.rb

```diff
@@ -1,3 +1,5 @@
+# encoding: utf-8
```

Comment:
no - not sure why that ended up in there -- leftover from debugging something I guess -- will remove